package.json

```diff
@@ -32,6 +32,7 @@
         "@sentry/react": "^7.114.0",
         "@tippyjs/react": "^4.2.6",
         "assert": "2.0.0",
+        "bignumber.js": "^9.1.2",
```

Comment:
@patricio0312rev I think the sdk already has a instance of big number, at lest some equivalente implementation

`import { BigNumber } from "@ardenthq/sdk-helpers";`

See arkvault for reference https://github.com/search?q=repo%3AArdentHQ%2Farkvault%20BigNumber&type=code